ports/atmel-samd/boards/circuitbrains_deluxe_m4/mpconfigboard.mk

```diff
@@ -0,0 +1,17 @@
+LD_FILE = boards/samd51x19-bootloader-external-flash.ld
```

Comment:
Obsolete; remove.

Reply:
Cleaned up board definitions per this conversation.